lib/amperize.js

```diff
@@ -170,46 +171,60 @@ Amperize.prototype.traverse = function traverse(data, html, done) {
      * @return {Object} element incl. width and height
      */
     function getImageSize(element) {
-      var options = url.parse(element.attribs.src),
-          timeout = 5000,
-          request = element.attribs.src.indexOf('https') === 0 ? https : http;
+      var imagePath = url.parse(element.attribs.src),
```

Comment:
For readability: This is not a path, this is an object.

Reply:
Makes sense!